lib/dataplane/kustomization.yaml

```diff
@@ -2,6 +2,14 @@
 apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
 
+secretGenerator:
```

Comment:
for such bits can rebase to main branch as have recent changes to fix these.

Reply:
Done